packages/bundler/src/Bundler.ts

```diff
@@ -41,7 +42,7 @@ export class Bundler implements IBundler {
      * @returns Promise<UserOpResponse>
      */
     async sendUserOp(userOp: UserOperation, chainId: ChainId): Promise<UserOpResponse> {
-        const hexifiedUserOp = deepHexlify(await resolveProperties(userOp))
+        const hexifiedUserOp = deepHexlify(await resolveProperties(userOp))        
         let params = [hexifiedUserOp, this.bundlerConfig.epAddress, chainId]
```

Comment:
I hope more methods will be added here like estimateUserOpGas, getUserOpByHash, getUserOpReceipt

Reply:
i can add them here but waiting for yash to implement them